src/tests/ftest/harness/pre_launch.py

```diff
@@ -0,0 +1,41 @@
+"""
+(C) Copyright 2021-2022 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import re
+
+from apricot import TestWithoutServers
+
+from run_utils import run_remote
+
+
+class HarnessPreLaunchTest(TestWithoutServers):
+    """Harness pre-launch test cases.
+
+    :avocado: recursive
+    """
+
+    def test_harness_pre_launch_users(self):
+        """Verify all expected users are setup correctly.
```

Comment:
This comment should include the name of the script that's supposed to have setup the users correctly, otherwise in years to come someone is going to have to go digging or looking in the git history to see what file was added/edited at the same time.

Reply:
Very good point